utils/mcmb/mcmb.c

```diff
@@ -3011,3 +3018,5 @@ static CMB_TRANSFORM_OP_FIND(*, cmb_mul_find)
 static CMB_TRANSFORM_OP_FIND(/, cmb_div_find)
 static CMB_TRANSFORM_OP_FIND(+, cmb_add_find)
 static CMB_TRANSFORM_OP_FIND(-, cmb_sub_find)
+
+// vim: filetype=c:tabstop=4:ai:expandtab
```

Comment:
Accident?

Reply:
Nope, that's a vi modeline.  They have to be somewhere within the first 5 or last 5 lines.  I didn't want it at the top messing up the same style we use everywhere else.